css/amp.css

```diff
@@ -176,6 +176,18 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   overflow: hidden !important;
 }
 
+/**
+ * TODO(dvoytenko, #19004): if/when "scroll-height-minheight" launched, merge
+ * these styles into _both_: `#i-amphtml-wrapper > body` and
+ * `html.i-amphtml-ios-embed-sd > body` sections above.
+ */
+.i-amphtml-body-minheight > body {
```

Comment:
Why not use `html.i-amphtml-ios-embed { min-height: ...` ?  It should avoid the BG color issue @aghassemi raised.

Reply:
That doesn't always help, unfortunately. If `body` height doesn't overflow, it also fails to scroll sometimes...

Reply:
adding a 1px by 101vh absolute visibility:hidden div to wrapper may be safer

Reply:
@aghassemi and I actually retested and apparently body is special-cased and the background applies to the whole document. So, stretching the body appears to be a better fix rather than a surrogate stretching element.